project/cmake/addons/addons/adsp.basic/adsp.basic.txt

```diff
@@ -1 +1 @@
-adsp.basic https://github.com/AchimTuran/adsp.basic master
+adsp.basic https://github.com/AchimTuran/adsp.basic fix_win32_cmake
```

Comment:
consider what happens the second time you build. for the clone of the git inside the buildsystem, "fix_win32_cmake" will resolve to the *local* branch named fix_win32_cmake and no updates will be applied. you should be using origin/fix_win32_cmake to have it pull in updates if the github repo changes.